buildSrc/src/main/kotlin/module.gradle.kts

```diff
@@ -128,6 +119,7 @@ fun Module.forceConfigurations() {
                     Coroutines.core,
                     Coroutines.bom,
                     Coroutines.coreJvm,
+                    "com.fasterxml.jackson.jr:jackson-jr-objects:2.15.3"
```

Comment:
Now that we know, we need it forced, let's extract this version to the respective `Jackson` object.